services/meminator-nodejs/src/tracing.ts

```diff
@@ -15,10 +13,18 @@ const sdk = new NodeSDK({
     traceExporter,
     // spanProcessors: [new ConfigurationSpanProcessor(), new BatchSpanProcessor(traceExporter)], // INSTRUMENTATION: report global configuration on every span
     instrumentations: [getNodeAutoInstrumentations(
-        // { '@opentelemetry/instrumentation-fs': { enabled: false } } // the fs tracing might be interesting here!
+        { '@opentelemetry/instrumentation-fs': { enabled: true } } // the fs tracing might be interesting here!
```

Comment:
Do we we want to re-enable fs? I thought we turned it off because it was too chatty?

Reply:
I just wanted to leave the functionality as it was. Since this line was previously commented out (different from all the other services), it seems fs instrumentation was enabled intentionally. So I uncommented it and set as enabled.